tests/fullstack-test/mpp/runtime_filter.test

```diff
@@ -0,0 +1,104 @@
+# Copyright 2023 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mysql> drop table if exists test.t1;
+mysql> create table test.t1(k1 int, t1_tinyint tinyint, t1_smallint smallint, t1_mediumint mediumint, t1_bigint bigint, t1_int int, t1_bit bit,  t1_bool bool, t1_float float, t1_double DOUBLE, t1_decimal decimal(10, 2));
+mysql> alter table test.t1 set tiflash replica 1;
+mysql> insert into test.t1 values (1,1,1,1,1111111,1, 1, 1, 1.0, 1.00, 1.0);
+mysql> insert into test.t1 values (1,1,1,1,1111111,1, 1, 1, 1.0, 1.00, 1.0);
+mysql> insert into test.t1 values (2,2,2,2,2222222,2, 0, 2, 2.0, 2.00, 2.0);
+mysql> insert into test.t1 values (3,2,2,2,2222222,2, 0, 2, 2.0, 2.00, 2.0);
+mysql> insert into test.t1 values (4,null,null,null,null,null, null, null, null, null, null);
+
+mysql> drop table if exists test.t2;
+mysql> create table test.t2(k1 int, t2_tinyint tinyint, t2_smallint smallint, t2_mediumint mediumint, t2_bigint bigint, t2_int int, t2_bit bit,  t2_bool bool, t2_float float, t2_double DOUBLE, t2_decimal decimal(10, 2));
+mysql> alter table test.t2 set tiflash replica 1;
+mysql> insert into test.t2 values (1,1,1,1,1111111,1, 1, 1, 1.0, 1.00, 1.0);
+mysql> insert into test.t2 values (1,1,1,1,1111111,1, 1, 1, 1.0, 1.00, 1.0);
+mysql> insert into test.t2 values (2,3,3,3,3333333,3, 1, 3, 3.0, 3.00, 3.0);
+mysql> insert into test.t2 values (3,3,3,3,3333333,3, 1, 3, 3.0, 3.00, 3.0);
+mysql> insert into test.t2 values (4,null,null,null,null,null, null, null, null, null, null);
+
+func> wait_table test t1
+func> wait_table test t2
+
+# inner join
+mysql> set @@tidb_isolation_read_engines='tiflash'; set tidb_enforce_mpp = 1; set tidb_runtime_filter_mode="LOCAL"; select t1_tinyint, t2_tinyint from test.t1, test.t2 where t1.t1_tinyint=t2.t2_tinyint;
```

Comment:
Please check the runtime filter has been enable by explain query plan before check the result

Reply:
The results of explain sql are not checked in fullstack test because the results are frequently affected by changes to tidb planner, resulting in ci instability.
Manually check that runtime filter takes effect in inner join and semi join.

Reply:
OK